pkg/client/clientset/versioned/fake/register.go

```diff
@@ -43,7 +43,7 @@ func init() {
 //
 //   import (
 //     "k8s.io/client-go/kubernetes"
-//     clientsetscheme "k8s.io/client-go/kuberentes/scheme"
+//     clientsetscheme "k8s.io/client-go/kubernetes/scheme"
```

Comment:
`pkg/client/clientset` is generated code. Does this come back after running `make codegen`?

Reply:
Interesting. I didn't know that. I'll check.

Reply:
It does indeed come back. Fixed.

Reply:
what will prevent it from coming back? was the codegen changed? If so, I must have overlooked it.

Reply:
This is still generated as "kuberentes". I just fixed the patch to not change this since it is generated.

Reply:
ok, makes sense to leave it.